maintainers/maintainer-list.nix

```diff
@@ -6706,6 +6706,12 @@
     githubId = 1104419;
     name = "Lucas Hoffmann";
   };
+  lunarequest = {
```

Comment:
Add yourself to maintainer list as separate commit with message `maintainers: add lunarequest`.